theano/printing.py

```diff
@@ -46,6 +46,9 @@
     except ImportError:
         # tests should not fail on optional dependency
         pydot_imported_msg = "Install the python package pydot or pydot-ng."
+    except Exception, e:
+        pydot_imported_msg = "An error happened while importing/triing pydot: "
```

Comment:
trying I amended the commit.

On Wed, Aug 31, 2016 at 3:55 PM, abergeron <notifications@github.com> wrote:

> In theano/printing.py
> <https://github.com/Theano/Theano/pull/4924#discussion_r77060990>:
>
> > @@ -46,6 +46,9 @@
> >      except ImportError:
> >          # tests should not fail on optional dependency
> >          pydot_imported_msg = "Install the python package pydot or pydot-ng."
> > +    except Exception, e:
> > +        pydot_imported_msg = "An error happened while importing/triing pydot: "
>
> What does "triing" means?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4924/files/4b9b85ff3bb14ad6ae0ffdf268c6832f4fd083de#r77060990>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALC-4PNP5R3Q7a9dmcrKk-B5OdVpOA1ks5qldw5gaJpZM4Jx7wZ>
> .
>